jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebInfConfiguration.java

```diff
@@ -504,13 +506,15 @@ public void makeTempDirectory(File parent, WebAppContext context)
             //if it is to be persisted, make sure it will be the same name
             //by not using File.createTempFile, which appends random digits
             tmpDir = new File(parent, temp);
+            configureTempDirectory(tmpDir, context);
```

Comment:
probably should now be called `ensureTempDirectory` ?

Reply:
This is a public API / method, don't want to change it for 9.4.x.
Be happy to change it for 10.0.x though.